cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/ProgramLifecycleHttpHandlerTest.java

```diff
@@ -790,13 +791,27 @@ public void testServices() throws Exception {
 
     Id.Service service1 = Id.Service.from(Id.Namespace.from(TEST_NAMESPACE1), APP_WITH_SERVICES_APP_ID,
                                           APP_WITH_SERVICES_SERVICE_NAME);
-    Id.Service service2 = Id.Service.from(Id.Namespace.from(TEST_NAMESPACE2), APP_WITH_SERVICES_APP_ID,
-                                          APP_WITH_SERVICES_SERVICE_NAME);
+    final Id.Service service2 = Id.Service.from(Id.Namespace.from(TEST_NAMESPACE2), APP_WITH_SERVICES_APP_ID,
+                                                APP_WITH_SERVICES_SERVICE_NAME);
+    HttpResponse activeResponse = getActiveResponse(service1);
```

Comment:
rename everything that has "active" in its name to "available"? Makes the code easier to read.